code/controllers/subsystem/garbage.dm

```diff
@@ -224,6 +226,8 @@ var/datum/subsystem/garbage_collector/SSgarbage
 // Return the appropriate QDEL_HINT; in most cases this is QDEL_HINT_QUEUE.
 /datum/proc/Destroy(force=FALSE)
 	tag = null
+	for(var/thing in SStimer.timer_src_dict[src])
```

Comment:
Thats a good point.... We should figure out if we want them to survive a change_turf(). i'm thinking maybe not.